scripts/dialog_xml2json.py

```diff
@@ -818,8 +819,8 @@ def convertAll(upperNodeJson, nodeXml):
         if not os.path.exists(getattr(config, 'common_outputs_directory')):
             os.makedirs(getattr(config, 'common_outputs_directory'))
             print('Created new output directory ' + getattr(config, 'common_outputs_directory'))
-        with open(os.path.join(getattr(config, 'common_outputs_directory'), getattr(config, 'common_outputs_dialogs')), 'w') as outputFile:
-            outputFile.write(json.dumps(dialogNodes, indent=4))
+        with io.open(os.path.join(getattr(config, 'common_outputs_directory'), getattr(config, 'common_outputs_dialogs')), 'w',encoding="utf-8") as outputFile:
+            outputFile.write(json.dumps(dialogNodes, indent=4,ensure_ascii=False,encoding="utf8"))
```

Comment:
I would prefer using single quotation marks and spaces after commas in function parameters to be consistent with the rest of the code..